symfony.lock

```diff
@@ -87,7 +87,7 @@
         "version": "v3.3.1"
     },
     "league/csv": {
-        "version": "9.8.0"
+        "version": "^9.8.0"
```

Comment:
@klaw772 what's the purpose of this? This breaks any subsequent `composer install` with an error about the caret not being valid.

I think `symfony.lock` is like `composer.lock`, you should never edit it manually.

Reply:
I had added the caret based on https://github.com/acquia/cli/pull/842#discussion_r824110292 - I can add a quick hotfix to remove it if it's causing a validation error. The `composer.json` includes the caret and this was autogenerated without it, so I can remove.

cc @grasmash

Reply:
Ah wait, I see https://github.com/acquia/cli/pull/845 now - thank you!

Reply:
FTR this is my fault. I was sure I had reviewed `composer.json`, not `symfony.lock`.